azurelinuxagent/distro/default/rdma.py

```diff
@@ -21,10 +21,14 @@
 
 import os
 import re
-import azurelinuxagent.logger as logger
+import azurelinuxagent.common.logger as logger
+import azurelinuxagent.common.utils.shellutil as shellutil
 
 
 class RDMAHandler(object):
+    def __init__(self):
+        self.driver_module_name = 'hv_network_direct'
```

Comment:
On 06/16/2016 07:11 PM, Ahmet Alp Balkan wrote:
>>  
>>  
>>  class RDMAHandler(object):
>> +    def __init__(self):
>> +        self.driver_module_name = 'hv_network_direct'
> 
> can we declare this just under `class RDMAHander`

Yes, it's a style choice. I generally prefer using __init__ but if you'd
rather me drop __init__ I certainly have no problem doint it the other
way. From a language perspective I am not certain if there is actually a
difference.